src/test/java/com/commercetools/project/sync/util/TestUtils.java

```diff
@@ -118,6 +121,21 @@ public static void assertCartDiscountSyncerLoggingEvents(
     assertSyncerLoggingEvents(syncerTestLogger, "CartDiscount", cartDiscountStatsSummary);
   }
 
+  public static void assertStateSyncerLoggingEvents(
+      @Nonnull final TestLogger syncerTestLogger, final int numberOfResources) {
+    final String stateSyncerStatsSummary =
+        format(
+            "Summary: %d state(s) were processed in total (%d created, 0 updated, 0 failed to sync and 0 state(s) with missing transition(s)).",
+            numberOfResources,
+            Math.max(
+                0,
+                numberOfResources
+                    - 1)); // 1 state is automatically built-in in all projects and processed, but
```

Comment:
in my machine this section was in 2 lines and I was confusing for me. Can you verify that it will be in a single line. Point is mathematical expression should be in a single line. comment can be splitted.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/133/commits/2daaba2cfd37c2660829c694a0c7a2c454f8730e